stream-chat-android-client/src/main/java/io/getstream/chat/android/client/extensions/StringExtensions.kt

```diff
@@ -22,4 +22,4 @@ internal fun String.camelCaseToSnakeCase(): String {
     return camelRegex.replace(this) { "_${it.value}" }.toLowerCase()
 }
 
-internal fun String.isAnonymousChannelId(): Boolean = startsWith("!members")
+internal fun String.isAnonymousChannelId(): Boolean = contains("!members")
```

Comment:
It doesn't always start with !members... if it is a cid, it is in the middle of the string.